src/algebraic_geometry/presheafed_space.lean

```diff
@@ -0,0 +1,173 @@
+-- Copyright (c) 2019 Scott Morrison. All rights reserved.
+-- Released under Apache 2.0 license as described in the file LICENSE.
+-- Authors: Scott Morrison
+import category_theory.instances.Top.presheaf
+
+universes v u
+
+open category_theory
+open category_theory.instances
+open category_theory.instances.Top
+open topological_space
+
+variables (C : Type u) [𝒞 : category.{v+1} C]
+include 𝒞
+
+namespace algebraic_geometry
+
+structure PresheafedSpace :=
+(X : Top.{v})
+(𝒪 : X.presheaf C)
+
+variables {C}
+
+namespace PresheafedSpace
+
+instance : has_coe_to_sort (PresheafedSpace.{v} C) :=
+{ S := Type v, coe := λ F, F.X.α }
+
+instance (F : PresheafedSpace.{v} C) : topological_space F := F.X.str
+
+structure hom (F G : PresheafedSpace.{v} C) :=
```

Comment:
Can these `hom`s be coerced into maps between the underlying spaces? Maybe via `forgetful_functor.map`?

Reply:
I am very wary of unnecessary coercions. What would this coercion be used for?